grpc-spring-boot-starter/src/main/java/org/lognet/springboot/grpc/GRpcServerRunner.java

```diff
@@ -97,6 +97,7 @@ private ServerServiceDefinition bindInterceptors(ServerServiceDefinition service
                 privateInterceptors)
                 .distinct()
                 .collect(Collectors.toList());
+        Collections.sort(interceptors,org.springframework.core.annotation.AnnotationAwareOrderComparator.INSTANCE);
```

Comment:
to be honest given the fact that grpc executes it's interceptor list back to front, we might as well reverse the ordered list here so that it's more intuitive for spring users. That way @Order(10) would be executed before @Order(20) which is what most people would expect.